src/Castle.Core.Tests/Castle.Core.Tests.csproj

```diff
@@ -48,33 +54,38 @@
 		<PackageReference Include="NLog" Version="4.5.0" />
 		<PackageReference Include="Serilog" Version="2.0.0" />
 		<PackageReference Include="Serilog.Sinks.TextWriter" Version="2.0.0" />
-		<PackageReference Include="System.Diagnostics.TraceSource" Version="4.3.0" />
-		<PackageReference Include="System.Reflection.TypeExtensions" Version="4.3.0" />
-		<PackageReference Include="System.Xml.ReaderWriter" Version="4.3.1" />
-		<PackageReference Include="System.Xml.XDocument" Version="4.3.0" />
-		<PackageReference Include="System.Xml.XmlDocument" Version="4.3.0" />
-		<PackageReference Include="System.Dynamic.Runtime" Version="4.3.0" />
-		<PackageReference Include="System.Reflection" Version="4.3.0" />
-		<PackageReference Include="System.Reflection.Emit" Version="4.3.0" />
-		<PackageReference Include="System.Collections.Specialized" Version="4.3.0" />
-		<PackageReference Include="System.ComponentModel" Version="4.3.0" />
-		<PackageReference Include="System.ComponentModel.TypeConverter" Version="4.3.0" />
-		<PackageReference Include="System.Runtime.CompilerServices.VisualC" Version="4.3.0" />
-		<PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
-		<PackageReference Include="System.Xml.XmlSerializer" Version="4.3.0" />
-		<PackageReference Include="System.Xml.XPath" Version="4.3.0" />
-		<PackageReference Include="System.Xml.XPath.XDocument" Version="4.3.0" />
-		<PackageReference Include="System.Xml.XPath.XmlDocument" Version="4.3.0" />
-		<ProjectReference Include="..\Castle.Core\Castle.Core.csproj" />
-		<ProjectReference Include="..\Castle.Services.Logging.log4netIntegration\Castle.Services.Logging.log4netIntegration.csproj" />
-		<ProjectReference Include="..\Castle.Services.Logging.NLogIntegration\Castle.Services.Logging.NLogIntegration.csproj" />
-		<ProjectReference Include="..\Castle.Services.Logging.SerilogIntegration\Castle.Services.Logging.SerilogIntegration.csproj" />
+    <ProjectReference Include="..\Castle.Core\Castle.Core.csproj" />
+    <ProjectReference Include="..\Castle.Services.Logging.log4netIntegration\Castle.Services.Logging.log4netIntegration.csproj" />
+    <ProjectReference Include="..\Castle.Services.Logging.NLogIntegration\Castle.Services.Logging.NLogIntegration.csproj" />
```

Comment:
Please fix these whitespace changes in this file (tabs, not spaces). I wouldn't have complained if it had been just the occasional few whitespace-only changes above, but this here seems quite messy. Also, it seems lots of `<PackageReference>`s were removed only to be added again a few lines further down. If you put them back in their original position we'd get a smaller commit diff.

Reply:
As already stated before, these changes follows YOUR settings in `.editorconfig`:
```
# Xml project files
[*.{csproj,vbproj,vcxproj,vcxproj.filters,proj,projitems,shproj}]
indent_style = space
indent_size = 2
```

The PackageReferences are moved from section for ALL targets to the target net461.
Github does not show this difference that correct.

Reply:
I am aware of the errors in `.editorconfig`&mdash;see my other review comment (below). I've noted that problem and am going to fix it. What would be much harder to find & fix is mixed-up whitespace strewn across the whole code base, thus my request that we at least stick to one type of indentation per file... the one that's currently used in most of the file.

I realise that debating whitespace is a little silly and aggravating as it's clearly less important in the grand scheme of things than what your PR is all about: adding the `netstandard2.x` targets to this project.

Please bear with me anyway, I'm just trying to ensure our code base stays as manageable as possible. :-)

>  The PackageReferences are moved from section for ALL targets to the target net461.
Github does not show this difference that correct.

I see. My mistake then. In that case, please disregard.